src/main/java/org/opentripplanner/middleware/otp/OtpRequestProcessor.java

```diff
@@ -76,6 +78,15 @@ private static String proxy(Request request, spark.Response response) {
         return otpDispatcherResponse.responseBody;
     }
 
+    /**
+     * Get request path intended for OTP API by removing the proxy endpoint (/otp).
+     */
+    static String stripFirstOtpPathPrefix(String uri) {
```

Comment:
This should be renamed `stripOtpProxyFromPath` or something similar.

Reply:
This method is irrelevant with renaming OTP_API_ROOT.